label_studio/ui_test/ui_test_label_studio.py

```diff
@@ -0,0 +1,435 @@
+
+    # Get all formats for given file type
+    # Get available templates for choosed type(jpg OR txt OR csv OR json ...)
+    def get_available_templates_for_choosed_type(self, choosedType):
+        availableTemplatesForChoosedType_arr = []
+        # Click for show "Advanced templates" section
+        self.driver.find_element_by_xpath(self.settings_advanced_templates_div).click()
+        #
+        # Click on first template
+        print("Clicking on first template for opening popup window")
+        # self.driver.find_element_by_css_selector(self.settings_first_avalaible_template).click()
+        settings_click_on_a = self.wait.until(EC.element_to_be_clickable((By.CSS_SELECTOR, self.settings_first_avalaible_template)))
+        settings_click_on_a.click() # !!!!!!!
+        #
+        # Clicking on popup's submit button
+        print('Clicking on "Proceed" button on the opened popup "Data that you\'ve put into textarea will be replaced, proceed?"')
+        settings_proceed_btn = self.wait.until(EC.element_to_be_clickable((By.XPATH, self.settings_proceed_btn)))
+        settings_proceed_btn.click()
+        time.sleep(2) # Waiting for popup window closing proces finished
+        #
+        # Get list of divs, each div contains all avaliable templates for that data type (jpg OR txt OR csv OR json)
+        self.settings_selectedTypeDiv_arr = self.driver.find_elements_by_xpath(self.settings_selectedTypeDiv)
+        # Get list of all div/i elements, meaning: get list of all template types
+        self.settings_selectedTypeNameFromDivi_arr = self.driver.find_elements_by_xpath(self.settings_selectedTypeNameFromDivi)
+
+        #print('Taking all template types')
+        for i in range(len(self.settings_selectedTypeNameFromDivi_arr)):
+            #<div..><i class="icon sound">...</i></div> ==> element i contains info about formate name (image, font, sound, ...)
+            element_i_class = self.settings_selectedTypeNameFromDivi_arr[i].get_attribute("class")
+            element_i_class = element_i_class[5:] #get "sound" from "icon sound"
+            if choosedType == element_i_class:
+                # All available templates for choosed type (jpg OR txt OR csv OR json)
+                availableTemplatesForChoosedType_arr = self.settings_selectedTypeDiv_arr[i].find_elements_by_xpath('.//div/a')
+        return availableTemplatesForChoosedType_arr
+
+
+        #
+        # print('*** all_templates_links:')
+        # all_templates_links = self.driver.find_elements_by_css_selector('a.use-template')
+        # # for link in all_templates_links:
+        # #     print(link)
+        # #     link.click()
+        # #
+        # for i in range(int(len(all_templates_links)/5)):
+        #     print(str(i)+'-->'+str(all_templates_links[i]))
+        #     if self.choosed_template == i:
+        #         all_templates_links[i].click()
+        #         break
+        # time.sleep(5)
+
+
+
+if __name__ == "__main__":
+    unittest.main()
+
```

Comment:
Need for CI scripts (github action yml) with clear description of input parameters

Reply:
Hi! what about this? Looking forward to getting
- launchable CI with parameters
- customization on xpath (since current master version is outdated, we need to adapt these scripts to existing pages somehow)